### PR TITLE
server: fix blurry PWA icon

### PR DESCRIPTION
**Pull request by quantum5**
I tried creating the server version as a PWA with Chrome on Android, but the icon is very blurry. My solution is to create a PNG of the icon and make the `manifest.webmanifest` use that instead of `favicon.ico`, which seems to have fixed the problem.

### CODE REVIEW

**Comment by rpg2014** [comment]
Thanks for this fix, this has been bugging me for a while. I see that chrome is complaining that a png, svg, or webp icons isn't specified in the manifest on my server's version (0.63.3). 
Image:
Chrome warning

I'm not super familiar with this project, but I made the same changes as the commit on my instance and it worked correctly.

**Review by rpg2014 — COMMENTED** [review]
Testing these changes locally and they work as expected.  Not 100% sure if this is the implementation the maintainers would want to go with.  I could see an argument around setting up a more generic express route that handles any files put into the public folder, rather than an route specific to `icon.png`

File: src/app.ts
Comment (rpg2014) [inline src/app.ts:50]: You should probably use the png that already exists in the images folder. https://github.com/TriliumNext/Notes/blob/develop/images/app-icons/png/512x512.png

Comment (quantum5) [reply src/app.ts:50]: Good find. I did a quick `find` and didn't see it, so I generated my own. But this is so much better.

**Review by quantum5 — COMMENTED** [review]
> Testing these changes locally and they work as expected. Not 100% sure if this is the implementation the maintainers would want to go with. I could see an argument around setting up a more generic express route that handles any files put into the public folder, rather than an route specific to `icon.png`

Yeah, I'll let the maintainers decide what the best way to serve that asset is. For now, I'll just get rid of the separate file.

File: src/app.ts
Comment (quantum5) [reply src/app.ts:50]: Ah I see, this file is only on the `develop` branch and not the last release or `master` branch.

**Review by eliandoran — APPROVED** [review]
LGTM.

**Pull request merged**
